### PR TITLE
docs: Added SslManagerService Target Filter parameter description

### PR DESCRIPTION
**Pull request by salvatore-coppola**
Added SslManagerService Target Filter parameter description

### CODE REVIEW

File: docs/gateway-configuration/web-console-configuration.md [hunk 1]
```diff
@@ -60,4 +60,9 @@ The default value is true
...
 
 Defines whether the "Certificate" authentication method is enabled or not
-The default value is true
\ No newline at end of file
+The default value is true
+
+## SslManagerService Target Filter
+
+It is possible to specify the SslManagerService containing the certificates truststore required to establish a new **https** connection, this is needed, for example, for fetching package descriptions from Eclipse Marketplace.
+
```
Comment (MMaiero) [inline docs/gateway-configuration/web-console-configuration.md:69]: Maybe we need to state the default? So that we match the other params?

Comment (MMaiero) [reply docs/gateway-configuration/web-console-configuration.md:69]: How that maps with upgrades?

Comment (salvatore-coppola) [reply docs/gateway-configuration/web-console-configuration.md:69]: the dafult value is the org.eclipse.kura.ssl.SslManagerService, I'll add to the description.

File: docs/gateway-configuration/web-console-configuration.md [hunk 2]
```diff
@@ -65,4 +65,5 @@ The default value is true
 ## SslManagerService Target Filter
 
 It is possible to specify the SslManagerService containing the certificates truststore required to establish a new **https** connection, this is needed, for example, for fetching package descriptions from Eclipse Marketplace.
+The default target is the default **org.eclipse.kura.ssl.SslManagerService**
```
Comment (MMaiero) [inline docs/gateway-configuration/web-console-configuration.md]: > The default target is the default

Maybe too many defaults in this sentence :-)

**Review by MMaiero — APPROVED** [review]

**Pull request merged**